WatchApp Extension/Controllers/OverrideSelectionController.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  OverrideSelectionController.swift
+//  WatchApp Extension
+//
+//  Created by Michael Pangburn on 1/31/19.
+//  Copyright © 2019 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+import WatchKit
+import LoopKit
+import LoopCore
+import WatchConnectivity
+
+
+protocol OverrideSelectionControllerDelegate: AnyObject {
+    func overrideSelectionController(_ controller: OverrideSelectionController, didSelectPreset preset: TemporaryScheduleOverridePreset)
+}
+
+
+final class OverrideSelectionController: WKInterfaceController, IdentifiableClass {
+
+    @IBOutlet private var table: WKInterfaceTable!
+
+    private let loopManager = ExtensionDelegate.shared().loopManager
```

Comment:
Watch architecture isn't very amenable to dependency injection, since we don't have control over instantiating the controllers ourselves. We could bundle up the presets in a class to pass in `awake(withContext:)` along with the delegate, but I'm not sure it's worth refactoring now.